fem/conduitdatacollection.cpp

```diff
@@ -639,6 +650,153 @@ ConduitDataCollection::BlueprintFieldToGridFunction(Mesh *mesh,
    return res;
 }
 
+//---------------------------------------------------------------------------//
+mfem::QuadratureFunction *
+ConduitDataCollection::BlueprintFieldToQuadratureFunction(Mesh *mesh,
+                                                          const Node &n_field,
+                                                          bool zero_copy)
+{
+   // n_conv holds converted data (when necessary for mfem api)
+   // if n_conv is used ( !n_conv.dtype().empty() ) we
+   // know that some data allocation was necessary, so we
+   // can't return a qf that zero copies the conduit data
+   Node n_conv;
+
+   const double *vals_ptr = NULL;
```

Comment:
Use `real_t` to support MFEM builds with different precisions (currently double or single only).